Update what-is-replay.md (#4776)
segmentio/segment-docs: src/guides/what-is-replay.md

Once a replay starts, you will not see replayed events in the Event Delivery tab.

> Replays can be initiated for some or all events but it is not possible to apply conditional filters that exclude certain rows of data from being replayed. You can set up [destination filters](/docs/connections/destinations/destination-filters/) to conditionally filter replayed events.

Reviewer: ```suggestion
You can initiate replays for some or all events, but you can't apply conditional filters that exclude certain rows of data from being replayed. You can set up [destination filters](/docs/connections/destinations/destination-filters/) to conditionally filter replayed events.
```
